### PR TITLE
Port break, dns, & leftpad to DaGdController

### PR DESCRIPTION
**Pull request by danopia**
Remaining `DaGdBase`:

* `DaGdHelpController` - will be replaced by newhelp
* `DaGdImageController` - it does some graphic stuff, possibly a more complex `DaGdController` port
* `DaGdCommanderController` - maybe we'll delete this?

### CODE REVIEW

File: src/applications/break/DaGdBreakController.php [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-final class DaGdBreakController extends DaGdBaseClass {
-  public function render() {
+final class DaGdBreakController extends DaGdController {
+  public function execute(DaGdResponse $response) {
     $environment = DaGdConfig::get('general.environment');
     if ($environment != 'development') {
-      error400('This page is disabled in the production environment.');
-      return false;
+      return $this->error(400,
+        'This page is disabled in the production environment.')->execute($response);
```
Comment (relrod) [inline src/applications/break/DaGdBreakController.php:8]:
```suggestion
      return $this
        ->error(400, 'This page is disabled in the production environment.')
        ->execute($response);
```

Just for consistency.

Comment (danopia) [reply src/applications/break/DaGdBreakController.php]: Is there a specific linter that would make these multi-lining decisions for this repo?

**Comment by relrod** [comment]
:shipit:

**Comment by relrod** [comment]
>    DaGdHelpController - will be replaced by newhelp

:pray: 

>    DaGdImageController - it does some graphic stuff, possibly a more complex DaGdController port

I don't think this will be _too_ bad. It needs to override `finalize()`, and probably return a `DaGdCacheableResponse` instance that sets the right header for the mimetype and so on. Should be doable. We'll have to use an in-memory file stream for the output of the image functions (e.g. `imagepng()`), since there doesn't seem to be any way to get them to return the raw data.

>    DaGdCommanderController - maybe we'll delete this?

yesplz.

**Pull request merged**